documentation/config_file_webserver.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
I think the prior documentation in this file was slightly out of date.

In our current Puppet Server 2.x packaging, we split the `webserver` and `web-router-service` sections into two different files - `webserver.conf` and `web-routes.conf`, respectively.

The default content of the `webserver.conf` file is:

~~~~hocon
webserver: {
    access-log-config = /etc/puppetlabs/puppetserver/request-logging.xml
    client-auth = want
    ssl-host = 0.0.0.0
    ssl-port = 8140
}
~~~~

The default content of the `web-routes.conf` file is:

~~~~hocon
web-router-service: {
    # These two should not be modified because the Puppet 4.x agent expects them to
    # be mounted at these specific paths
    "puppetlabs.services.ca.certificate-authority-service/certificate-authority-service": "/puppet-ca"
    "puppetlabs.services.master.master-service/master-service": "/puppet"
    "puppetlabs.services.legacy-routes.legacy-routes-service/legacy-routes-service": ""

    # This controls the mount point for the puppet admin API.
    "puppetlabs.services.puppet-admin.puppet-admin-service/puppet-admin-service": "/puppet-admin-api"
}
~~~~

I'm okay with having this split done as part of this PR or as a follow-on one, whichever you prefer.